administrator/templates/atum/component.php

```diff
@@ -8,7 +8,9 @@
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
 
+// phpcs:disable PSR1.Files.SideEffects
 defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
```

Comment:
```suggestion
defined('_JEXEC') or die;
```
PHPCS would also be happy without that.

Reply:
Removed the tags